frontend/src/app/app-routing.module.ts

```diff
@@ -3,7 +3,7 @@ import { RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
   { path: '', loadChildren: () => import('./modules/workspace/workspace.module').then(m => m.WorkspaceModule) },
-  { path: 'register', loadChildren: () => import('./modules/auth/auth.module').then(m => m.AuthModule) }
+  { path: 'recodo', loadChildren: () => import('./modules/auth/auth.module').then(m => m.AuthModule) }
```

Comment:
Can you make login -> /login, register -> /register, workspace -> /personal or what we have there